receiver/postgresqlreceiver/scraper_test.go

```diff
@@ -38,6 +38,7 @@ func TestScraper(t *testing.T) {
 	cfg := createDefaultConfig().(*Config)
 	cfg.Databases = []string{"otel"}
 	cfg.Metrics.PostgresqlDeadlocks.Enabled = true
+	cfg.Metrics.PostgresqlTempFiles.Enabled = true
```

Comment:
Do we have any test to make sure this metric isn't included by default, or when it's disabled?

Reply:
You meant to say, whether we have a test to probe default is set or not?

Reply:
Yes, thank you, that's what I meant!

Reply:
No problem! I have pushed a new commit with added tests. Please let me know if you think we should replicated for all scenarios ( multiple db ). Also added deadlock into the fray since it is the other optional metric that has been recently added.

Reply:
Thanks for adding!

Reply:
wait, I had a problem with the last commit. This should fix all the lint errors.